lib/core/Canvas.spec.ts

```diff
@@ -158,7 +158,7 @@ canvas.removeConnection(connectionLike);
 
 canvas.resized();
 
-canvas.scroll({ x: 100, y: 100 });
```

Comment:
Not sure how that passed the type check. We'd want to ensure that either `dx` _or_ `dy` are set.

Reply:
It previously passed because delta was typed incorrectly as Point.
Dunno if checking at runtime would be needed as it seems to default to some acceptable values in case the use isn't setting dx/dy.